azext_iot/operations/central.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=wrong-import-order,too-many-lines
+
+import asyncio
```

Comment:
A top level import of asyncio is not going to work. We use asyncio functionality for Python 3.5+ ....by adding import here the module is no longer compatible with Python 2.7.

Reply:
Ah, thanks for letting me know.  How is it used in the events library? Is it restricted to commands with the python 3.5+ check?

Reply:
Yea thats right. The pattern is to check that we have a compatible runtime (3.5+) then import the constrained elements such as `asyncio` and `uamqp`.

Reply:
> Yea thats right. The pattern is to check that we have a compatible runtime (3.5+) then import the constrained elements such as `asyncio` and `uamqp`.

Hmm interesting, I'll move stuff around so it works as expected then.  Thanks~!